### PR TITLE
add updated docs for how to enable elevated browser permissions

### PR DESCRIPTION
**Pull request by LeoMcA**

### CODE REVIEW

File: docs/browser_permissions.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+# Elevated Browser Permissions
+
+support.mozilla.org has certain elevated permissions in Firefox,
```
Comment (akatsoulas) [inline docs/browser_permissions.md:3]: I would rephrase this to focus on what these permissions do and not that sumo has elevated perms.

Comment (LeoMcA) [reply docs/browser_permissions.md:3]: I added a clause to the intro explaining what the permissions do, how's that?

File: docs/browser_permissions.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+## Testing locally
+
+Firefox only grants permissions to `https` origins,
+so we need to serve our local instance over `https`.
+The easiest way to do this is to set up a reverse proxy,
+with a tool like [`local-ssl-proxy`](https://github.com/cameronhunter/local-ssl-proxy):
+
+```
+npm install -g local-ssl-proxy
```
Comment (akatsoulas) [inline docs/browser_permissions.md:62]: There's a django extension that wraps the dev server. Maybe we can bundle that in the development setup and use a control flag to activate

File: docs/browser_permissions.md (hunk 1 above)
Comment (akatsoulas) [reply docs/browser_permissions.md:3]: I would still rephrase it a bit, on how to interact with the browser and not about the perms that the platform has

**Review by akatsoulas — APPROVED** [review]
r+wc
